contracts/token-distribution/SaleRounds.sol

```diff
@@ -93,106 +115,39 @@ contract SaleRounds is TokenDistribution, GameOwner, ERC20 {
-        address socialWalletAddress = walletAddresses[2];
-        address teamWalletAdrdress = walletAddresses[3];
-        address treasuryWalletAddress = walletAddresses[4];
-
-        //ALLOCATIONS WITH WALLET CONSTANT
-        reserveTokensInternal(RoundType.PLAYANDEARN, playAndEarnWalletAddress, playAndEarnDistribution.supply);
-        reserveTokensInternal(RoundType.SOCIAL, socialWalletAddress, socialDistribution.supply);
-        reserveTokensInternal(RoundType.TEAM, teamWalletAdrdress, teamDistribution.supply);
-        reserveTokensInternal(RoundType.TREASURY, treasuryWalletAddress, treasuryDistribution.supply);
-
-        //NO VESTING TIME SO DIRECT MINTING -- PUBLIC IS NOT SCOPED HERE
-        _mint(exhangesWalletAddress, exchangesDistribution.supply);
-    }
-
-    modifier isEligibleToReserveToken(string calldata _roundType) {
-        RoundType roundType = getRoundTypeByKey(_roundType);
-
-        require(roundType != RoundType.PUBLIC, "reservation is not supported for this round");
-        require(isGameOwnerAddress() || addressMap[roundType][_msgSender()] == true, "address is not confirmed to reserv the token");
-        _;
-    }
-
-    modifier isRoundActive(string calldata _roundType) {
-        RoundType roundType = getRoundTypeByKey(_roundType);
 
-        require(activeRound[roundType] == true , "round is not active");
-        _;
+        initialReserveAndMint(_walletAddresses);
     }
 
-    modifier isInvestRound(string calldata _roundType) {
-        RoundType roundType = getRoundTypeByKey(_roundType);
+    function addAddressForDistribution(string calldata _roundType, address _address) external
+        onlyGameOwner returns(bool) {
 
-        require(roundType == RoundType.SEED ||
-        roundType == RoundType.PRIVATE ||
-            roundType == RoundType.PUBLIC , "round is not invest round");
-        _;
-    }
-
-    modifier claimableRound(string calldata _roundType) {
-        RoundType roundType = getRoundTypeByKey(_roundType);
-
-        require(roundType != RoundType.PUBLIC && roundType != RoundType.EXCHANGES && roundType != RoundType.ADVISOR, "Claiming is not supported for this round");
-        _;
-    }
-
-    function setActiveRound(string calldata _roundType) public onlyGameOwner {
         RoundType roundType = getRoundTypeByKey(_roundType);
-        require(activeRound[roundType] == false, "Round is already active");
-        setActiveRoundInternally(roundType);
-    }
-
-    function setActiveRoundInternally(RoundType _roundType) private {
-        activeRound[_roundType] = true;
-        roundDistribution[_roundType].startTime = block.timestamp;
-    }
-
-    function addAddressForDistribution(string calldata _roundType, address _address) public
-        onlyGameOwner isRoundActive(_roundType) returns(bool) {
-
-        RoundType roundType = getRoundTypeByKey(_roundType);
-        addressMap[roundType][_address] = true;
         addressList[roundType].push(_address);
```

Comment:
Wow this one became a really short one, but what does this address do? 
Is it still just a whitelist? Is this needed?
Might be a bug; does the unit test to test claiming succeed for all claiming groups?